### PR TITLE
Updated setup documentation on Nextcloud integration

### PR DESCRIPTION
**Pull request by marcalcobe**
The complete documentation on how to setup your Nextcloud documentation using the OAuth values step by step.
Also created the user-guide documentation md where I will be working next.

@psatyal could you please review the documentation from a language perspective?
@wielinde could you please review that the step by step is clear? Do we want to test this docs with someone totally disconnected from the Nextcloud integration design and development to see if it works as intended?

https://community.openproject.org/projects/openproject/work_packages/41752/activity

### CODE REVIEW

File: docs/system-admin-guide/integrations/nextcloud/README.md [hunk 1]
```diff
@@ -1,79 +1,101 @@
...
-  title: Nextcloud integration
+  title: Nextcloud integration setup
   priority: 600
-description: Nextcloud and OpenProject integration
+description: Nextcloud integration setup
 keywords: integrations, apps, Nextcloud
 ---
 
-# OpenProject and Nextcloud integration 
+<<<<<<< Updated upstream
```
Comment (psatyal) [inline docs/system-admin-guide/integrations/nextcloud/README.md]: These lines with <<<< are a result of errors related to GitHub fetching/merging, I think.

**Review by psatyal — COMMENTED** [review]
Hi @marcalcobe. I've reviewed the system doc and changed phrasing in a few places to make the text clearer and sometimes more direct, restyled all references to buttons or actions in bold and changed "IMPORTANT" bits to quoted style (to respect current styling). Could you review to make sure that my re-structuring of certain sentences does not change the meaning/details of the integration?

I did not review the user guide bit yet because it seems like it's still work in progress.

File: docs/system-admin-guide/integrations/nextcloud/README.md [hunk 2]
```diff
@@ -6,85 +6,81 @@ description: Nextcloud integration setup
...
 
 ![Nextcloud_host](3_2_01 -NC_Step_1.png)
 
-Click on the "Save" button and open a new tab with your OpenPoject instance.
+Click on the **Save** button and open your OpenProject instance in a new tab.
 
 **2. Introduce the basic information of your integration** - in OpenProject
 
-As an administrator, in the new browser tab please open the File storages from the Administration page. To start with the configuration of your new Nextcloud integration click on the "+" button to add a new storage.
+Make sure you are logged in as an administrator in your OpenProject instance. Navigate to *File storages* from the Administration page. To start configuring new Nextcloud integration, click on the **Add** (+) button to add a new file storage.
```
Comment (marcalcobe) [inline docs/system-admin-guide/integrations/nextcloud/README.md]: Hi @psatyal,
I think a "your" is missing after the "configuring".

File: docs/system-admin-guide/integrations/nextcloud/README.md [hunk 3]
```diff
@@ -6,85 +6,81 @@ description: Nextcloud integration setup
...
 
 **4. Generate and copy the Nextcloud OAuth values** - in Nextcloud and OpenProject
 
-As in the previous step the OAuth values are generated automatically, but this time the Nextcloud OAuth are created in the step 3 of the Nextcloud OpenProject Integration settings. This values are the ones that will allow the connection From Nextcloud to OpenProject, therefore, again, you need to copy them but this time from Nextcloud to OpenProject (without closing the Nextcloud one). If you haven't clicked on the "Done. Continue setup" button from the previous step, now you can click it to see the screen where you will be able to paste the Nextcloud OAuth values.
+As in the previous step, the OAuth values are generated automatically, but this time on the Nextcloud end. These are needed to allow the connection From Nextcloud to OpenProject to function, so you will need to once again copy them. Copy these values from Nextcloud to OpenProject (without closing the Nextcloud tab). If you had not clicked on the *Done. Continue setup* button from in previous step, you can now do so to see the screen where you will be able to paste the Nextcloud OAuth values in OpenProject.
```
Comment (marcalcobe) [inline docs/system-admin-guide/integrations/nextcloud/README.md]: Hi @psatyal,
In the last sentence after *Done. Continue setup* button, is it correct "from in previous step"? Shouldn't it be something like "in the previous step"?

File: docs/system-admin-guide/integrations/nextcloud/README.md [hunk 4]
```diff
@@ -6,85 +6,81 @@ description: Nextcloud integration setup
...
 
-Now the integration setup is complete and ready to use, but as administrator you still need to activate the file storages module and specify in which projects you need the file storage integration that you just set up.
+Now that the integration setup is complete and ready to use, there is just one more step for you the administrator: you need to activate the *File storages* module and specify for which projects you would like to enable your new Nextcloud integration.
 
-To activate the module please go to the Project settings in the desired project and access the Modules menu entry in the side menu, there you can activate the module "File storages". Once this is done a new lateral menu entry called "File storages" at the bottom of the menu where you can select which file storage integration you want to activate in this project.
+To activate the module, go to the *Project settings* in the desired project and access the Modules menu entry in the side menu. There, activate the "File storages" module. Once this is done, a new menu entry called "File storages" appears at the bottom, where you can select your new Nextcloud integration.
```
Comment (marcalcobe) [inline docs/system-admin-guide/integrations/nextcloud/README.md]: Hi @psatyal,

I have the feeling that removing the part of the menu makes the sentence unclear (the bottom of the content? bottom of the complete screen?) I would like to keep the menu or the side menu there to make it clear.

**Review by marcalcobe — COMMENTED** [review]
Hi @psatyal,
Thank you so much for the changes on the phrasing. I included some small details that I'm not sure we should change from a text perspective.

**Review by psatyal — APPROVED** [review]

**Review by psatyal — APPROVED** [review]

File: docs/system-admin-guide/integrations/nextcloud/README.md (hunk 4 above)
Comment (psatyal) [reply docs/system-admin-guide/integrations/nextcloud/README.md]: Hi @marcalcobe, just fixed this in a separate commit.

**Comment by psatyal** [comment]
Hi @marcalcobe. Made changes based on your comments and approved them on my end.

File: docs/system-admin-guide/integrations/nextcloud/README.md [hunk 5]
```diff
@@ -1,79 +1,96 @@
...
-description: Nextcloud and OpenProject integration
+description: Nextcloud integration setup
 keywords: integrations, apps, Nextcloud
 ---
 
-# OpenProject and Nextcloud integration 
+# OpenProject and Nextcloud integration setup 
 
-The OpenProject and Nextcloud integration will improve the productivity of their enterprise users. It combines the strengths of the market leading content collaboration platform Nextcloud and the leading open source project management software OpenProject.
+The integration between OpenProject and Nextcloud has the potential to improve the productivity of all users. It combines the strengths of Nextcloud, the market-leading collaboration platform, and OpenProject, the leading open source project management software.
```
Comment (wielinde) [inline docs/system-admin-guide/integrations/nextcloud/README.md]: Hmm, "market leading collaboration platform". Isn't OpenProject a collaboration platform, too? Shouldn't we focus on the file sharing/serving/hosting bit of Nextcloud?

File: docs/system-admin-guide/integrations/nextcloud/README.md [hunk 6]
```diff
@@ -1,79 +1,96 @@
...
 
-The integration is available starting with Nextcloud 20. It enables users to keep an eye on ongoing project activities directly in their Nextcloud instance.
+The integration enables users to keep an eye on ongoing project activities directly in their Nextcloud instance and link their Nextcloud files to OpenProject work packages. To learn more about how to use the integration, please refer to the [Nextcloud user guide](../../../user-guide/nextcloud-integration/)
```
Comment (wielinde) [inline docs/system-admin-guide/integrations/nextcloud/README.md]: If  with "keep an eye on the ongoing project activities" you mean the dashboard, then I would deprioritize that aspect and focus more on "Where are the files that I need to be able to complete a task?" and "Where do I put the final files after completing a task?".

File: docs/system-admin-guide/integrations/nextcloud/README.md [hunk 7]
```diff
@@ -1,79 +1,96 @@
...
 
-## Benefits of the integration
+## Step-by-step setup instructions
 
-The first step of the combined effort is the integration of OpenProject in the Nextcloud dashboard. Users can add an OpenProject widget to display latest changes to project's work packages. With this, it offers users a view of ongoing projects and activities.
+The integration is available starting with Nextcloud 20 and OpenProject 12.2. To be able to configure this integration, you need to be an administrator in both Nextcloud and OpenProject instances.
```
Comment (wielinde) [inline docs/system-admin-guide/integrations/nextcloud/README.md]: For integrating with OpenProject 12.2 the minimal version of Nextcloud is 22. The minimal version version of the Nextcloud app is 2.0.0

File: docs/system-admin-guide/integrations/nextcloud/README.md [hunk 8]
```diff
@@ -1,79 +1,96 @@
...
 
-## Step by step instructions
+**1. Add OpenProject integration app and connect the instance** - in Nextcloud
```
Comment (wielinde) [inline docs/system-admin-guide/integrations/nextcloud/README.md]:
```suggestion
**1. Add the "OpenProject integration" app and connect the instance** - in Nextcloud
```

File: docs/system-admin-guide/integrations/nextcloud/README.md [hunk 9]
```diff
@@ -1,79 +1,96 @@
...
 
-The integration is available starting with Nextcloud 20. It enables users to keep an eye on ongoing project activities directly out of their Nextcloud instance.
+To activate your integration to OpenProject in Nextcloud, navigate to the built-in app store under your user name in _Your apps_. You can use the search field in the top right corner to look for OpenProject integration app. Click the button **Download and enable**.
```
Comment (wielinde) [inline docs/system-admin-guide/integrations/nextcloud/README.md]: What do you mean with "under your user name"? Do you mean the menu that opens when you click your user avatar on the top right corner in Nextcloud?

File: docs/system-admin-guide/integrations/nextcloud/README.md [hunk 10]
```diff
@@ -1,79 +1,96 @@
...
 
-**Add OpenProject integration app**
+![Nextcloud_app_store](Nextcloud_app_store.png)
 
-To activate your integration to OpenProject in Nextcloud, navigate to the built in app store under your user name in Your apps. You can use the search field in the top right corner to look for the OpenProject integration. Click the button Download and enable.
+Once the OpenProject integration app is downloaded and enabled, you can access it through the side menu and enter the host URL of your desired OpenProject instance.
```
Comment (wielinde) [inline docs/system-admin-guide/integrations/nextcloud/README.md]: Might be important to mention that you find it within the settings.

File: docs/system-admin-guide/integrations/nextcloud/README.md (hunk 5 above)
Comment (marcalcobe) [reply docs/system-admin-guide/integrations/nextcloud/README.md]: Hi @wielinde. This. is related with how we describe the integration everywhere. Let's discuss it with @birthe so we are all in the same page and I will update it afterwards.

File: docs/system-admin-guide/integrations/nextcloud/README.md (hunk 6 above)
Comment (marcalcobe) [reply docs/system-admin-guide/integrations/nextcloud/README.md]: Hi @wielinde. Sounds good to me. Also might change after we agree on a common description.

File: docs/system-admin-guide/integrations/nextcloud/README.md (hunk 7 above)
Comment (marcalcobe) [reply docs/system-admin-guide/integrations/nextcloud/README.md]: I will update this information also in this documentation page.

File: docs/system-admin-guide/integrations/nextcloud/README.md (hunk 9 above)
Comment (marcalcobe) [reply docs/system-admin-guide/integrations/nextcloud/README.md]: Hi @wielinde. I didn't touch this step as it was coming from the previous documentation but i updated it now.

File: docs/system-admin-guide/integrations/nextcloud/README.md (hunk 10 above)
Comment (marcalcobe) [reply docs/system-admin-guide/integrations/nextcloud/README.md]: I will add this to the documentation as its much clearer.

File: docs/system-admin-guide/integrations/nextcloud/README.md (hunk 5 above)
Comment (psatyal) [reply docs/system-admin-guide/integrations/nextcloud/README.md]: Agreed, I'd recommend the same phrasing as the other doc: "It combines the strengths of [Nextcloud](https://nextcloud.com/), the world’s most-deployed on-premises collaborative file storage platform and [OpenProject](https://www.openproject.org/), the leading free and open source project management and collaboration software."

File: docs/system-admin-guide/integrations/nextcloud/README.md (hunk 6 above)
Comment (psatyal) [reply docs/system-admin-guide/integrations/nextcloud/README.md]: Agreed, will update.

File: docs/system-admin-guide/integrations/nextcloud/README.md (hunk 7 above)
Comment (psatyal) [reply docs/system-admin-guide/integrations/nextcloud/README.md]: I'll add this information

File: docs/system-admin-guide/integrations/nextcloud/README.md (hunk 9 above)
Comment (psatyal) [reply docs/system-admin-guide/integrations/nextcloud/README.md]: Adding the word "menu" to make avoid potential confusion.

File: docs/system-admin-guide/integrations/nextcloud/README.md (hunk 10 above)
Comment (psatyal) [reply docs/system-admin-guide/integrations/nextcloud/README.md]: Added this precision.

**Comment by psatyal** [comment]
Updated the text based on @wielinde's comments.

**Review by psatyal — APPROVED** [review]

**Review by marcalcobe — COMMENTED** [review]
All the changes has been done on the text. The only pending text would be the unified description of the integration in case is needed on the top of the documentation page.
Also a review from any @opf/doc-writers would be appreciated as double check.

File: docs/system-admin-guide/integrations/nextcloud/README.md [hunk 11]
```diff
@@ -1,79 +1,96 @@
...
-description: Nextcloud and OpenProject integration
+description: Nextcloud integration setup
 keywords: integrations, apps, Nextcloud
 ---
 
-# OpenProject and Nextcloud integration 
+# OpenProject and Nextcloud integration setup 
 
-The OpenProject and Nextcloud integration will improve the productivity of their enterprise users. It combines the strengths of the market leading content collaboration platform Nextcloud and the leading open source project management software OpenProject.
+The integration between OpenProject and Nextcloud has the potential to improve the productivity of all users. It combines the strengths of Nextcloud, the world’s most deployed on-premises collaborative file storage platform and OpenProject, the leading free and open source project management and collaboration software
```
Comment (wielinde) [inline docs/system-admin-guide/integrations/nextcloud/README.md]: @marcalcobe @psatyal Tiny detail: What is the right genitive tick in English? The one used here is not the one I have on my US English keyboard. Seems suspicious to me. 

```suggestion
The integration between OpenProject and Nextcloud has the potential to improve the productivity of all users. It combines the strengths of Nextcloud, the world's most deployed on-premises collaborative file storage platform and OpenProject, the leading free and open source project management and collaboration software
```

Comment (psatyal) [reply docs/system-admin-guide/integrations/nextcloud/README.md]: @wielinde Indeed, the one you've replaced it with is more standard; don't know where the original came from. Commiting. (For reference: I have an ANSI keyboard to with an "ABC - Extended" setup). (Edited)

File: docs/system-admin-guide/integrations/nextcloud/README.md [hunk 12]
```diff
@@ -1,79 +1,96 @@
...
+This integration allows you to link files and folders in Nextcloud with work packages in OpenProject, allowing you to see all files related to a work package (in Open Project) and all work packages related to a file (in Nextcloud). As a project member, you no longer need to lose time trying to find the right files to be able to complete your tasks, and the integration of OpenProject-specific notifications on Nextcloud dashboard ensures that no change goes unnoticed. 
 
-## Benefits of the integration
+## Step-by-step setup instructions
 
-The first step of the combined effort is the integration of OpenProject in the Nextcloud dashboard. Users can add an OpenProject widget to display latest changes to project's work packages. With this, it offers users a view of ongoing projects and activities.
+For integrating with OpenProject 12.2, the minimum required version of Nextcloud is 22. For the desktop Nextcloud application, the minimum supported version is 2.0.0. To be able to configure this integration, you need to have administrator privileges in both your Nextcloud and OpenProject instances.
```
Comment (wielinde) [inline docs/system-admin-guide/integrations/nextcloud/README.md]: AFAIK Nextcloud calls their plugins "Apps" and not "applications". 

```suggestion
For integrating with OpenProject 12.2, the minimum required version of Nextcloud is 22. For the Nextcloud app  "OpenProject integration" that extends your Nextcloud installation, the minimum required version is 2.0.0. To be able to configure this integration, you need to have administrator privileges in both your Nextcloud and OpenProject instances.
```

Comment (psatyal) [reply docs/system-admin-guide/integrations/nextcloud/README.md]: Ah, good point @wielinde. I would remove the comma after 

> integration",

Looks like I can modify your suggested change, let me try it.

File: docs/system-admin-guide/integrations/nextcloud/README.md [hunk 13]
```diff
@@ -1,79 +1,96 @@
...
 
-## Step by step instructions
+**1. Add the "OpenProject integration" app and connect the instance** - in Nextcloud
 
-The integration is available starting with Nextcloud 20. It enables users to keep an eye on ongoing project activities directly out of their Nextcloud instance.
+To activate your integration to OpenProject in Nextcloud, navigate to the built-in app store in the menu under your user name by clicking on _Your apps_. You can use the search field in the top right corner to look for the OpenProject integration app. Click the button **Download and enable**.
```
Comment (wielinde) [inline docs/system-admin-guide/integrations/nextcloud/README.md]: There is no user name in the Nextcloud header. There is only an avatar.

```suggestion
To activate your integration to OpenProject in Nextcloud, navigate to the built-in app store. You find the app store in the menu that opens when you click your round user avatar in the top right corner. In the app store click on _Integrations_. There you will find the "OpenProject integration" app. You can also use the search field in the top right corner to look for the OpenProject integration app. Once you have found the app, click the button **Download and enable**.
```

File: docs/system-admin-guide/integrations/nextcloud/README.md [hunk 14]
```diff
@@ -1,79 +1,96 @@
...
 
-**Add OpenProject integration app**
+![Nextcloud_app_store](Nextcloud_app_store.png)
 
-To activate your integration to OpenProject in Nextcloud, navigate to the built in app store under your user name in Your apps. You can use the search field in the top right corner to look for the OpenProject integration. Click the button Download and enable.
+Once the OpenProject integration app is downloaded and enabled, you can access it from within the Settings page, via the the side menu. Start by entering the *host URL* of your desired OpenProject instance.
```
Comment (wielinde) [inline docs/system-admin-guide/integrations/nextcloud/README.md]:
```suggestion
Once the OpenProject integration app is downloaded and enabled, you can access it from within the _Settings_ page, via the side menu. Start by entering the _Host URL_ of your desired OpenProject instance.
```

File: docs/system-admin-guide/integrations/nextcloud/README.md [hunk 15]
```diff
@@ -1,79 +1,96 @@
...
 
-![Nextcloud_app_store](Nextcloud_app_store.png)
+![Nextcloud_host](3_2_01 -NC_Step_1.png)
+
+Click on the **Save** button and open your OpenProject instance in a new tab.
```
Comment (wielinde) [inline docs/system-admin-guide/integrations/nextcloud/README.md]:
```suggestion
Click on the **Save** button and open your OpenProject instance in a new browser tab.
```

File: docs/system-admin-guide/integrations/nextcloud/README.md [hunk 16]
```diff
@@ -1,79 +1,96 @@
...
+
+**2. Introduce the basic information of your integration** - in OpenProject
+
+Make sure you are logged in as an administrator in your OpenProject instance. Navigate to *File storages* from the Administration page. To start configuring your new Nextcloud integration, click on the **Add (+)** button to add a new file storage.
+
+![Storage_index](3_0_00-OP_OAuth_Empty_Index.png)
+
+By default, the storage *provider type* is set to Nextcloud; this does not need to be modified. To proceed, enter a *name* to your Nextcloud storage. This will be visible to all the users using it. We highly recommended choosing a clear and distinct name that allows users to differentiate it from other potential file storages integrations in the future. You will also need to set the *Host URL* of your Nextcloud instance, including _https://_ bit of the address.
```
Comment (wielinde) [inline docs/system-admin-guide/integrations/nextcloud/README.md]:
```suggestion
By default, the storage *Provider type* is set to _Nextcloud_; this does not need to be modified. To proceed, enter a *Name* to your Nextcloud storage. This name will be visible to all the users using it. We highly recommended choosing a clear and distinct name that allows users to differentiate it from other potential file storages integrations in the future. You will also need to set the *Host URL* of your Nextcloud instance. The *Host URL* is the web address of your Nextcloud instance. Make sure you include the _https://_ bit of the address.
```

File: docs/system-admin-guide/integrations/nextcloud/README.md [hunk 17]
```diff
@@ -1,79 +1,96 @@
...
+
+![Storage_basic_information](3_0_01-OP_General_Info.png)
+
+Once you fill in the mandatory fields, click on the **Save and continue setup** button.
```
Comment (wielinde) [inline docs/system-admin-guide/integrations/nextcloud/README.md]:
```suggestion
Once you filled in all mandatory fields, click on the **Save and continue setup** button.
```

File: docs/system-admin-guide/integrations/nextcloud/README.md [hunk 18]
```diff
@@ -1,79 +1,96 @@
...
+
+**3. Generate and copy the OpenProject OAuth values** - in OpenProject and Nextcloud
+
+In this step, the *OpenProject OAuth values* are generated automatically. These values are needed to permit OpenProject to connect to Nextcloud. You will need to copy them from the Nextcloud integration window in a new tab (without exiting or closing your OpenProject tab) and pasting them in the respective fields under _OpenProject OAuth settings_.
```
Comment (wielinde) [inline docs/system-admin-guide/integrations/nextcloud/README.md]:
```suggestion
In this step, the *OpenProject OAuth values* are generated automatically. These values are needed to permit OpenProject to connect to Nextcloud. If you have not done so yet, we recommend you to best have two browser tabs open, the current one with OpenProject and the former one with Nextcloud. Now copy the two *OpenProject OAuth values* from the OpenProject tab to the respective fields in the Nextcloud tab. Simply paste them into the respective _OpenProject OAuth settings_ fields.
```

File: docs/system-admin-guide/integrations/nextcloud/README.md [hunk 19]
```diff
@@ -1,79 +1,96 @@
...
 
-**Activate the OpenProject integration app**
+> **Important**: The *OpenProjects OAuth values (client ID and client secret)* are not accessible again after you close the window. Please make sure you copy the generated values you see in the _Nextcloud OpenProject Integration settings_.
```
Comment (wielinde) [inline docs/system-admin-guide/integrations/nextcloud/README.md]:
```suggestion
> **Important**: The *OpenProjects OAuth values (client ID and client secret)* are not accessible again after you close the window. Please make sure you copy the generated values you see in the _Nextcloud OpenProject Integration settings_. Treat these values with care. Do not save them in any public place. Simply treat them as carefully as you would treat an important password.
```

File: docs/system-admin-guide/integrations/nextcloud/README.md [hunk 20]
```diff
@@ -1,79 +1,96 @@
...
+![Nextcloud_OP_OAuth_values](3_2_03-NC_Step_2.png)
 
-Enter the URL of your OpenProject instance and your access token which you can find in OpenProject under My Account and then Access token. Reset the API token and copy/paste it.
+Once you have copied the values, click on the **Save** button to proceed to the next step in Nextcloud.
 
-![OpenProject_API_key](OpenProject_API_key.png)
+**4. Generate and copy the Nextcloud OAuth values** - in Nextcloud and OpenProject
 
-![OpenProject_API_key_copy](OpenProject_API_key_copy.png)
+As in the previous step, the *OAuth values* are generated automatically, but this time on the Nextcloud end. These values are needed to allow the connection from Nextcloud to OpenProject, so you will need to once again copy them here and paste them in OpenProject (without closing the Nextcloud tab). If you had not clicked the **Done. Continue setup** button in the previous step, you can do so now to proceed to the screen where you will be able to paste the Nextcloud OAuth values in OpenProject.
```
Comment (wielinde) [inline docs/system-admin-guide/integrations/nextcloud/README.md]:
```suggestion
As in the previous step, the *OAuth values* are generated automatically, but this time on the Nextcloud end. These values are needed to allow the connection from Nextcloud to OpenProject, so you will need to once again copy them here and paste them in OpenProject (without closing the Nextcloud tab). If you had not clicked the **Done. Continue setup** button in the OpenProject tab in the previous step, you can do so now to proceed to the screen where you will be able to paste the Nextcloud OAuth values in OpenProject.
```

File: docs/system-admin-guide/integrations/nextcloud/README.md [hunk 21]
```diff
@@ -1,79 +1,96 @@
...
+![OpenProject_success](3_4_00 - OAuth - OP Success.png)
 
-In your personal settings in Connected accounts, please remember to also activate the Enable navigation link to display a link to your OpenProject instance in the header navigation.
+![Nextcloud_success](3_2_05-NC_Success.png)
 
-![Nextcloud_connected_account](Nextcloud_connected_account.png)
+**5. Enable the file storages module and activate it in the desired project** - in OpenProject
 
-The link will show here:
+Now that the integration setup is complete and ready to use, there is just one more step for you as administrator: you need to activate the *File storages* module and specify for which projects you would like to enable your new Nextcloud integration.
```
Comment (wielinde) [inline docs/system-admin-guide/integrations/nextcloud/README.md]:
```suggestion
Now that the integration setup is complete and ready to use, there is just one more step for you as administrator: in every project in which you want to use the integration you need to activate the *File storages* module and add your recently configured file storage to the project.
```

File: docs/system-admin-guide/integrations/nextcloud/README.md [hunk 22]
```diff
@@ -1,79 +1,96 @@
...
-## Where do I find the Nextcloud integration in OpenProject?
+![openproject_reset_OAuth](3_4_03-OP_Replace_Alert.png)
 
-Further integration efforts are under way, which will deliver a Nextcloud integration also on the OpenProject side.
 
-## What if project notifications are not displayed?
+## How to delete a file storage integration
 
-If the notifications are not displayed in your Nextcloud dashboard, please check the following in your Nextcloud basic settings: in the background jobs, Cron must be activated.
+As an administrator, you can always delete the integration to Nextcloud using the **Delete** button in OpenProject settings or the **Reset** button in Nextcloud settings.
```
Comment (wielinde) [inline docs/system-admin-guide/integrations/nextcloud/README.md]:
```suggestion
As an administrator, you can always delete the integration to Nextcloud using the **Delete** button in  the OpenProject settings or the **Reset** button in the Nextcloud settings.
```

File: docs/system-admin-guide/integrations/nextcloud/README.md [hunk 23]
```diff
@@ -1,79 +1,96 @@
...
 
-![Cron_job_settings](Cron_job_settings.png)
\ No newline at end of file
+> **Important:** If you perform this action, the integration will be reset and deleted and all settings and user connections that were created will be deleted. This means that should you want to reconfigure the integration, you will need complete the entire setup process once again.
```
Comment (wielinde) [inline docs/system-admin-guide/integrations/nextcloud/README.md:96]: @marcalcobe Hmm, the delete in OpenProject deletes all file links. The reset button in Nextcloud just deletes the connections but not the links. We may want to be very clear what does what. So we better invest these lines here.

File: docs/system-admin-guide/integrations/nextcloud/README.md [hunk 24]
```diff
@@ -1,79 +1,96 @@
...
+Now that the integration setup is complete and ready to use, there is just one more step for you as administrator: you need to activate the *File storages* module and specify for which projects you would like to enable your new Nextcloud integration.
 
-![Navigation_link_OpenProject](Navigation_link_OpenProject.png)
+To activate the module, go to the **Project settings** in the desired project and access the **Modules** entry in the side menu. There, activate the *File storages* module. Once this is done, a new menu entry called **File storages** appears at the bottom of the side menu, where you can select your new Nextcloud integration.
 
-By activating "enable unified search for tickets" in your personal settings, the Nextcloud dashboard will include OpenProject information in the the built-in universal search:
+![module_activation](Settings_modules.png)
 
-![Unified_search](Unified_search.png)
+![settings_files_storages](Settings_files_storages.png)
```
Comment (wielinde) [inline docs/system-admin-guide/integrations/nextcloud/README.md:77]: @marcalcobe Here we could link to the user guide, in the sense of "All further steps are explained in the User guide"

**Review by wielinde — COMMENTED** [review]
Hey @marcalcobe I added comments and suggestions. Very often I added more details and text to reduce ambiguity and confusion. Please use whatever you feel is an improvement.

File: docs/system-admin-guide/integrations/nextcloud/README.md (hunk 17 above)
Comment (psatyal) [reply docs/system-admin-guide/integrations/nextcloud/README.md]: Just a note here: "fill in" is gramattically correct, you do not use the past tense with when the sentence begins with "once (something something). (I don't know why, actually 🤔)... So we should revert if commited :) @marcalcobe, @wielinde.

File: docs/system-admin-guide/integrations/nextcloud/README.md (hunk 18 above)
Comment (psatyal) [reply docs/system-admin-guide/integrations/nextcloud/README.md]: "We recommend you have" (vs. "to best have").

File: docs/system-admin-guide/integrations/nextcloud/README.md (hunk 23 above)
Comment (marcalcobe) [reply docs/system-admin-guide/integrations/nextcloud/README.md:96]: @wielinde totally agree with this quote add. Thanks.

**Review by psatyal — APPROVED** [review]

File: docs/system-admin-guide/integrations/nextcloud/README.md (hunk 24 above)
Comment (marcalcobe) [reply docs/system-admin-guide/integrations/nextcloud/README.md:77]: Hi @wielinde, @psatyal added this information in his last commit.

**Comment by marcalcobe** [comment]
@opf/doc-writers, @birthe or @as-op all the relevant changes and reviews has been done from our side. Can you please do the last double check before we merge it to 12.2?

**Review by as-op — CHANGES_REQUESTED** [review]
* There are spaces in the image filenames, which we unfortunately need to avoid:
"3_2_01 -NC_Step_1.png"
"3_2_06 -NC_OP_OAuth_Replace.png"
"3_4_00 - OAuth - OP Success.png"

* Please delete the "_old" image folder. (Old images are always available through git)

File: docs/system-admin-guide/integrations/nextcloud/README.md [hunk 25]
```diff
@@ -1,79 +1,102 @@
...
 
-Further integration efforts are under way, which will deliver a Nextcloud integration also on the OpenProject side.
+> **Important:** If you perform this action, the integration will be reset and deleted and all settings and user connections that were created will be deleted. This means that should you want to reconfigure the integration, you will need complete the entire setup process once again.
 
-## What if project notifications are not displayed?
+## Using the integration
 
-If the notifications are not displayed in your Nextcloud dashboard, please check the following in your Nextcloud basic settings: in the background jobs, Cron must be activated.
+Once the file storage is added and enabled for projects, your users are able to take full advantage of the Nextcloud-OpenProject integration. For more information on how link Nextcloud files to work packages in OpenProject and access linked work packages in Nextcloud, please refer to the [Nextcloud integration user guide](../../../user-guide/nextcloud-integration/).
+.
```
Comment (as-op) [inline docs/system-admin-guide/integrations/nextcloud/README.md]: There's an unhappily single dot. :)

Comment (marcalcobe) [reply docs/system-admin-guide/integrations/nextcloud/README.md]: The two things are already solved. Thanks @as-op

**Review by as-op — APPROVED** [review]

**Review by as-op — CHANGES_REQUESTED** [review]
Sorry, one more issue. Just noticed it. In this PR there is also a stub for the [user-guide/nextcloud-integration/README.md](https://github.com/opf/openproject/blob/c64c52b16869210d685e45fa2094fd939b8fb52c/docs/user-guide/nextcloud-integration/README.md).
But it's also topic in PR https://github.com/opf/openproject/pull/11016 - [README.md](https://github.com/opf/openproject/blob/7279d1f7270e464c1c247c61244cddf8900374c9/docs/user-guide/nextcloud-integration/README.md).
Since it's a new file in both PR, it's possible and best to delete it in this PR to avoid a merge conflict.
Thanks!

**Comment by marcalcobe** [comment]
Hi @as-op,
I've deleted the file from here.

**Review by as-op — APPROVED** [review]

**Comment by as-op** [comment]
Recreated & merged to release/12.2 at https://github.com/opf/openproject/pull/11062

**Pull request closed without merging**